batch/2.0/_index.md

```diff
@@ -36,19 +36,20 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-08-27 with the following results.
 
 | Representative                                 | Representative for: | Vote |
 |------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
-| Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
-
-The ballot was run in the [jakarta.ee-spec mailing list]()
+| Kenji Kazumura                                 | Fujitsu             | +1   |
+| Dan Bandera, Kevin Sutter                      | IBM                 | +1   |
+| Ed Bratt, Dmitry Kornilov                      | Oracle              | +1   |
+| Andrew Pielage, Matt Gill                      | Payara              | +1   |
+| Scott Stark, Mark Little                       | Red Hat             | +1   |
+| David Blevins, Jean-Louis Monteiro             | Tomitribe           | +1   |
+| Ivar Grimstad                                  | EE4J PMC            | +1   |
+| Marcelo Ancelmo, Martijn Verburg               | Participant Members | +1   |
+| Werner Keil                                    | Committer Members   | +1   |
+| Scott (Congquan) Wang                          | Enterprise Members  | no vote cast |
+|                                                | Total               |  9   |
+
+The ballot was run in the [jakarta.ee-spec mailing list](https://www.eclipse.org/lists/jakarta.ee-spec/msg00727.html)
```

Comment:
The EE 8 Platform Spec. points to the summary message. I used that as my template. If that's correct, this should point to: https://www.eclipse.org/lists/jakarta.ee-spec/msg00789.html

Reply:
I'm actually fine either way since you can follow the trail from either end.